packages/babel-helpers/src/helpers.js

```diff
@@ -325,7 +325,7 @@ helpers.objectSpread = helper("7.0.0-beta.0")`
       var source = (arguments[i] != null) ? Object(arguments[i]) : {};
       var ownKeys = Object.keys(source);
       if (typeof Object.getOwnPropertySymbols === 'function') {
-        ownKeys = ownKeys.concat(Object.getOwnPropertySymbols(source).filter(function(sym) {
+        Array.prototype.push.apply(ownKeys, Object.getOwnPropertySymbols(source).filter(function(sym) {
```

Comment:
Since this is a runtime helper, we could use `ownKeys.push.apply` so that it's shorted (and `ownKeys` can be minified to a single char).

Reply:
Do you mean `ownKeys.push(...)`?